lite/examples/pose_estimation/android/README.md

```diff
@@ -0,0 +1,64 @@
+# TensorFlow Lite Pose Estimation Android Demo
+
+### Overview
+This is an app that continuously detects the body parts in the frames seen by
+ your device's camera. These instructions walk you through building and running
+ the demo on an Android device. Camera captures are discarded immediately after
+ use, nothing is stored or saved.
+ 
+ The app demonstrates how to use 3 models:
+ * Posenet
+ * Movenet Lightning
+ * Movenet Thunder
+ 
+ See this [blog post](https://blog.tensorflow.org/2021/05/next-generation-pose-detection-with-movenet-and-tensorflowjs.html) for a comparison between these models.
```

Comment:
It might be nice to at least say a few words about when to use Posenet vs Lightning vs Thunder. The blog post discusses Lightning vs Thunder, but it doesn't mention PoseNet. Neither do the TFHub Lightning/Thunder models reference PoseNet. We should speak to when a user would want to use one vs the other (potential benefits/tradeoffs).

Reply:
We'll address it in a separated TFLite Movenet blog post.